controllers/ControllerAdmin.php

```diff
@@ -20,49 +21,52 @@ public function __construct($method, $twig, $params){
 		$this->twig = $twig;
 		$this->params = $params;
 		$this->userManager = new AuthManager();
-
-		if (method_exists(ControllerAdmin::class, $method) ) {
-			$this->$method();
+		$target = $method[2];
+		if (method_exists(ControllerAdmin::class, $target) ) {
+			$this->$target();
 		}else{
 			echo $this->twig->render('404.twig');
 		}
 
 	}
 
 	public function profile(){
+		$key = (new Session)->getKey('user');
+
 		echo $this->twig->render('profile.twig', ['logged' => self::LOGGED,
-		                                          'user'   => $_SESSION['user']['pseudo'],
-		                                          'value'  => $_SESSION['user']['email'],
-												   'avatar'=> $_SESSION['user']['avatar']]);
+		                                          'user'   => $key['pseudo'],
+		                                          'value'  => $key['email'],
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$key['email']'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)